pkg/task/task.go

```diff
@@ -56,3 +61,11 @@ func (t *Task) Created() time.Time {
 	}
 	return time.Unix(u.Time().UnixTime())
 }
+
+func (t *Task) LastState() DatedTaskState {
```

Comment:
This could be named just `State()` - I think it is clear that this is the current/latest state.